hub.util.exceptions.TransformError
==================================

.. currentmodule:: hub.util.exceptions

.. autoexception:: TransformError